site/_posts/2017-03-06-deletions-and-dragons.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Last Week in ManageIQ: Deletions and Dragons"
+author: d-m-u
+date: 2017-03-07
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+## The Players 
+[Jamlamin Tarmikos][jt]: Male Human Dwarven Defender Str 12 Int 15 Wis 18 Dex 12 Con 16 Cha 17  
+[Thernda Strifelaugher][tw]: Female Halfling Dwarven Defender Str 15 Int 15 Wis 13 Dex 13 Con 6 Cha 11  
+[Janmorel][jc]: Male Half-Orc Loremaster Str 7 Int 15 Wis 12 Dex 7 Con 3 Cha 12  
+[Nerisstina Cupshigh][nc]: Female Gnome Paladin Str 14 Int 15 Wis 10 Dex 8 Con 13 Cha 9  
+[Krisnys Bearcharger][kb]: Female Elf Shadowdancer Str 4 Int 15 Wis 8 Dex 10 Con 5 Cha 19  
+[Eillyassa Chorster][ec]: Female Gnome Ranger Str 15 Int 15 Wis 3 Dex 3 Con 13 Cha 3  
+[Jannys Falconsflight][jf]: Female Half-Orc Paladin Str 6 Int 15 Wis 5 Dex 7 Con 11 Cha 4  
+[Janella Silentall][jr]: Female Half-Orc Dwarven Defender Str 11 Int 15 Wis 8 Dex 18 Con 11 Cha 13  
+[Magnys Gladomain][mh]: Female Elf Rogue Str 10 Int 15 Wis 13 Dex 10 Con 9 Cha 15  
+
+## Week Nine Overview
+
+Hello! Welcome to The Route of Elemental Code Deletions, a campaign where I, [Drew](https://github.com/d-m-u), am DMing a company of players as they attempt to navigate the treacherous module of Week Nine.
```

Comment:
I had to look up `DMing` which upon realising I meant Dungeon Mastering seems pretty obvious now given this weeks theme, but maybe we should use the full words.